eng/jacoco-test-coverage/pom.xml

```diff
@@ -91,30 +91,35 @@
       <artifactId>azure-core-tracing-opentelemetry</artifactId>
       <version>1.0.0-beta.5</version> <!-- {x-version-update;com.azure:azure-core-tracing-opentelemetry;current} -->
     </dependency>
+    <dependency>
+      <groupId>com.azure</groupId>
+      <artifactId>azure-cosmos</artifactId>
+      <version>4.0.1-beta.4</version> <!-- {x-version-update;com.azure:azure-cosmos;current} -->
+    </dependency>
     <dependency>
       <groupId>com.azure</groupId>
       <artifactId>azure-data-appconfiguration</artifactId>
       <version>1.2.0-beta.1</version> <!-- {x-version-update;com.azure:azure-data-appconfiguration;current} -->
     </dependency>
     <dependency>
       <groupId>com.azure</groupId>
-      <artifactId>azure-identity</artifactId>
-      <version>1.1.0-beta.5</version> <!-- {x-version-update;com.azure:azure-identity;current} -->
+      <artifactId>azure-data-schemaregistry</artifactId>
```

Comment:
Most of these are alphabetise changes where I cut and paste the dependencies into the right alphbetical order. The one that matters is line 116 - 117.

Reply:
This is great! Thank you!